packages/@react-spectrum/card/src/CardView.tsx

```diff
@@ -77,6 +71,7 @@ function CardView<T extends object>(props: SpectrumCardViewProps<T>, ref: DOMRef
 
   let state = useGridState({
     ...props,
+    selectionMode: cardOrientation === 'horizontal' && layoutType === 'grid' ? 'none' : props.selectionMode,
```

Comment:
why both horizontal and grid? i think no horizontal cards support selection?
also brings up an interesting idea, should we console warn for unsupported combinations of cards and layouts?
i see below that you just ignore the user sometimes https://github.com/adobe/react-spectrum/pull/2481/files#diff-01da5939b3c7131a7950777a23d1a3a30e621f34e9ef00d4a6b4533644fee4deR194
do we want to be fully protected or never protected? is an in-between option confusing?

Reply:
I'm fine with adding a console warning for unsupported combinations, just unclear on what those combinations will end up being since the designs are still kinda up in the air :(. Thats why I made it specifically horizontal + grid rather than just horizontal

I'm leaning towards being fully protected, right now the other layouts don't even support cardOrientation so those layout won't be affected even if the user passes horizontal as an option